doc/src/tutorials/creating-packages.md

```diff
@@ -275,13 +275,17 @@ test-specific dependencies, are available, see below.
 
 ### Test-specific dependencies
 
-There are two ways of adding test-specific dependencies (dependencies that are not dependencies of the package but will still be available to
-load when the package is tested).
+There are two ways of adding test-specific dependencies (dependencies that are not dependencies of the package but
+will still be available to load when the package is tested). If no `test/Project.toml` exists Pkg will default to
+use the `target` based test specific dependencies.
 
 #### `target` based test specific dependencies
 
-Using this method of adding test-specific dependencies, the packages are added under an `[extras]` section and to a test target,
-e.g. to add `Markdown` and `Test` as test dependencies, add the following to the `Project.toml` file:
+!!! compat
+    This older method of adding test-specific dependencies will be supported for all Julia 1.X releases.
```

Comment:
This note implies that `test/Project.toml` is the new replacement, which it may have intended to be when introduced, but that has weakened. See https://github.com/JuliaLang/Pkg.jl/pull/3574

Reply:
Mhh i didn't mean to imply that. I just took the language as is. Would striking the older be sufficient?
How do you feel about:
>  This method of adding test-specific dependencies works and will continue to work on all Julia 1.X releases.

Reply:
Sorry, my bad. I guess the updated language is still not great in my Pkg PR given I thought it was the old language here 😂